charts/scheduler-service/values.yaml

```diff
@@ -160,6 +160,8 @@ curl:
       runAsUser: 100
       readOnlyRootFilesystem: true
     # it does not make sense to have probes for curl
+    # resources for curl if needed
+    resources: {}
```

Comment:
Can you please provide recommended or reference values for resources? We are using the following comment for other resources in values:

```    
    # We follow recommendations and general guidelines to manage resources from:
    # https://master.sdk.operatorframework.io/docs/best-practices/managing-resources/
    # We recommend default requests for CPU and Memory, leaving the limits as a conscious
    # choice for the user. If you do want to specify your own resources, uncomment the following
    # lines, adjust them as necessary, and remove the curly braces after 'resources'.
    # requests:
    #   cpu: 70m
    #   memory: 1100M
    # limits: {}
```